src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,37 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, selectedTab, onTabSelected }) => {
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => {
+            const isTabSelected = cn({
+              'is-active': selectedTab.id === tab.id,
+            });
+
+            return (
+              <li key={tab.id} className={isTabSelected} data-cy="Tab">
```

Comment:
it would be better to place each attribute on a separate line

Reply:
> it would be better to place each attribute on a separate line

Good point

Reply:
The linter automatically puts it on one line